src/test/functional/utlis/helpers.ts

```diff
@@ -15,4 +15,26 @@ export class FunctionalTestHelpers {
     const updatedNumFieldsets = await I.countElement(fieldsetSelector);
     expect(updatedNumFieldsets - numFieldsets).equal(1);
   }
+
+  public static async clearFieldsets(containerId: string, buttonName: string) {
+    const fieldsetSelector = `${containerId} fieldset`;
+    const fieldsetCount = await I.countElement(fieldsetSelector);
+    // Remove all fieldsets except the empty new one and the template
+    for (let i = fieldsetCount; i > 2; i--) {
+      const deleteButtonSelector = `${containerId} button[name="${buttonName}"]`;
+      const elementExist = await I.checkElement(deleteButtonSelector);
+      expect(elementExist).equal(true);
+      await I.click(deleteButtonSelector);
```

Comment:
Could you call your new clickSaveButton method here?

Reply:
I have changed it to call 'save' directly from the helper method